src/ParallelRun.jl

```diff
@@ -136,21 +141,21 @@ function infer_init(rcfs::Vector{RunCamcolField},
     tic()
     catalog = SDSSIO.read_photoobj_files(rcfs, stagedir,
                         duplicate_policy=duplicate_policy)
-    timing.read_photoobj = toq()
-    Log.info("$(length(catalog)) primary sources")
+    timing.read_photoobj = timing.read_photoobj + toq()
+    #ntputs(grank(), threadid(), "$(length(catalog)) primary sources")
```

Comment:
Are you worried the logging statements will print so much that they slow down the job? These print just once per box--not very often.